io-sim-classes/io-sim-classes.cabal

```diff
@@ -18,6 +18,11 @@ source-repository head
   location: https://github.com/input-output-hk/ouroboros-network
   subdir:   io-sim-classes
 
+flag checktvarinvariant
```

Comment:
When/where do we enable this flag?

Thinking out loud: we want to enable it *always* (?) for `test-consensus` and `test-storage`. We don't want to have to pass it manually via the cmd line.